packages/jellyfish-transaction-builder/src/txn/txn_builder_governance.ts

```diff
@@ -45,13 +45,13 @@ export class TxnBuilderGovernance extends P2WPKHTxnBuilder {
     }
     let creationFee = new BigNumber('5')
     if (this.network.name === 'mainnet') {
-      creationFee = new BigNumber('100')
+      creationFee = new BigNumber('50')
     }
     if (this.network.name === 'testnet') {
       creationFee = new BigNumber('50')
     }
     if (createVoc.options === 1) {
-      creationFee = new BigNumber('10000')
+      creationFee = new BigNumber('5000')
```

Comment:
Hi @kuegi some test is failing, can you help keep this `10000` for testnet, and only set `5000` for mainnet? thanks!

Reply:
on testnet, the 10000 was false before already: 
<img width="393" alt="image" src="https://github.com/BirthdayResearch/jellyfishsdk/assets/29012906/22587552-3704-4349-b3df-524ab1239482">
longterm, ocean should provide the govs and this should take it directly from the chain. But IMHo now its most important to get this fixed for users. Cause noone is able to create a VOC via the LW.

Reply:
@lykalabrada what network/networkparams are the tests running on?

Reply:
adapted it so that testnet is correct too now (but still hardcoded) and regtest in tests keep working for now.